spk/syncthing/src/ca_reloader.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+CERT_DIR=/usr/local/etc/certificate/syncthing/syncthing-webui
+CONF_DIR=/var/packages/syncthing/var
+if [ ! -d "$CONF_DIR" ]; then
+   CONF_DIR=/var/packages/syncthing/target/var
+fi
+SYNCTHING=/var/packages/syncthing/target/bin/syncthing
+
+case $1 in
+    syncthing_webui)
+        # Forcefully overwrite certificate files by symlinks
+        ln -sf $CERT_DIR/cert.pem $CONF_DIR/https-cert.pem
+        ln -sf $CERT_DIR/privkey.pem $CONF_DIR/https-key.pem
+
+        # Required: set $HOME environment variable
+        HOME=$CONF_DIR
+        export HOME
+
+        $SYNCTHING cli --home=$CONF_DIR operations restart
```

Comment:
Have you tried `synopkg restart`? Just to be sure?

Reply:
Yes, it works when run as `root`, but fails if run as user `sc-syncthing`.  As presumed in the original PR description.

Reply:
Right, maybe good to mark this off in the original PR description? I was using that as a guide of the changes and consideration in this PR.

Reply:
Done.